### PR TITLE
Add Flippy: pygame Reversi clone

### PR DESCRIPTION
**Pull request by takluyver**
With @asweigart's agreement, I've packaged one of the games from his book [Make your own computer games with Python & Pygame](http://inventwithpython.com/pygame/). This follows my recent work to add a shared module for pygame. While this is intended to serve as an example for other people packaging pygame games, it's also a real, playable game suitable for passing a few idle minutes.

I haven't prepared the Appstream metadata yet; I'll add that later.

### CODE REVIEW

**Comment by nedrichards** [comment]
bot, build com.inventwithpython.flippy

**Comment by nedrichards** [comment]
https://flathub.org/builds/#/builders/2/builds/2698

Fails at 'no appstream metadata' obviously - but that means that when you do add it - should be good.

**Comment by takluyver** [comment]
OK, I've added the appstream metadata. The one thing I can't add from [the requirements](https://github.com/flathub/flathub/wiki/App-Requirements#appstream) is a releases section - because this is an example for a book, it doesn't have releases or versions, as far as I know (hopefully @asweigart can correct me if I'm wrong).

I also don't see an explicit open source license, but if Al names one, we could add the ` ⁠<project_license/>` tag.

**Comment by nedrichards** [comment]
If there isn't an open source licence then it's proprietary by default, which is fine - but it's important to put that there..

**Comment by nedrichards** [comment]
bot, build com.inventwithpython.flippy

**Comment by takluyver** [comment]
[Successful build](https://flathub.org/builds/#/builders/2/builds/2715).

I'll try to get Al's input on the license; I understand that legally the absence of a license means it's proprietary, but I suspect that's not really the intention.

**Comment by asweigart** [comment]
Yes, this game is released under a BSD 3-clause license. Feel free to make use of it! Thanks!

**Comment by takluyver** [comment]
Thanks Al, I've added the license tag in Appstream. As far as I'm aware, this is ready now. :-)

File: com.inventwithpython.flippy.json [hunk 1]
```diff
@@ -0,0 +1,63 @@
+{
+    "id": "com.inventwithpython.flippy",
+    "runtime": "org.freedesktop.Platform",
+    "sdk": "org.freedesktop.Sdk",
+    "runtime-version": "1.6",
+    "finish-args": [
+       "--socket=x11",
+       "--command=/app/bin/flippy"
```
Comment (barthalion) [inline com.inventwithpython.flippy.json]: Use top level `command` instead.

File: com.inventwithpython.flippy.json [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+        {
+            "name": "flippy",
+            "sources": [
+                {
+                  "type": "archive",
+                  "url": "http://invpy.com/flippyimages.zip",
+                  "sha256": "09ca8543237b42b152e5256e7c996cbff9ca124534abba0cd915fa1c11fe2af2"
+                }
+            ],
+            "buildsystem": "simple",
```
Comment (barthalion) [inline com.inventwithpython.flippy.json]: Please move `buildsystem` here (and the one below) above `sources`.

**Comment by takluyver** [comment]
Thanks, done.

**Comment by nedrichards** [comment]
bot, build com.inventwithpython.flippy

**Comment by takluyver** [comment]
Looks like the buildbot is having network issues: https://flathub.org/builds/#/builders/2/builds/2793

**Comment by asweigart** [comment]
Thanks for adding it!

**Comment by nedrichards** [comment]
Repository has been created: https://github.com/flathub/com.inventwithpython.flippy

**Comment by takluyver** [comment]
Thanks @nedrichards ! If it's not already the case, can I ask for @asweigart to be invited as a collaborator as well? :-)

**Comment by nedrichards** [comment]
Already the case, but thanks for asking!

**Pull request closed without merging**